tests/e2e/core-tests/specs/merchant/wp-admin-order-customer-payment-page.test.js

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable jest/no-export, jest/no-disabled-tests */
+import {createSimpleProduct} from "@woocommerce/e2e-utils";
+
+/**
+ * Internal dependencies
+ */
+const {
+	merchant,
+	createSimpleOrder,
+	addProductToOrder,
+} = require( '@woocommerce/e2e-utils' );
+
+const config = require( 'config' );
+const simpleProductName = config.get( 'products.simple.name' );
+const simpleProductPrice = config.has( 'products.simple.price' ) ? config.get( 'products.simple.price' ) : '9.99';
+
+let orderId;
+
+const runMerchantOrdersCustomerPaymentPage = () => {
+	describe('WooCommerce Merchant Flow: Orders > Customer Payment Page', () => {
+		beforeAll(async () => {
+			await merchant.login();
+			await createSimpleProduct();
+			orderId = await createSimpleOrder();
+			await addProductToOrder( orderId, simpleProductName );
+		});
+
+		it('should be able to view the customer order page and complete payment', async () => {
```

Comment:
I debated on if this would would be a good one to split out like that, and I agree with that. I'll go ahead and update the tests so it's four individual `it()` steps 👍

Reply:
@rrennick I ended up breaking up the tests to be more granular, as below:

```
  WooCommerce Merchant Flow: Orders > Customer Payment Page
    ✓ should show the customer payment page link on a pending payment order (1121ms)
    ✓ should load the customer payment page (892ms)
    ✓ can pay for the order through the customer payment page (1138ms)
```

I just wanted to confirm real quick your thoughts on the `from customer order page` as one of the `it()` flows. Is this for the customer loading it from the My Account page? Or did you have something else in mind?

Thanks!

Reply:
> Is this for the customer loading it from the My Account page?

Yeah, I was forgetting the distinction between merchant and shopper flows. The three `it()` calls are what I was suggesting.